wp-content/civi-extensions/goonjcustom/goonjcustom.php

```diff
@@ -166,6 +166,7 @@ function goonjcustom_civicrm_pageRun( &$page ) {
 					var urlParams = new URLSearchParams(settings.url);
 					var activityTypeId57 = "57";
 					var activityTypeId61 = "61";
+					var activityViewTypeId61 = "61";
```

Comment:
Do we need these two variables: `activityTypeId61` & `activityViewTypeId61`

Reply:
Yes, we need this because I use it to hide certain fields later. If I were using activityTypeId61, it would also hide fields that I don't want to hide. That's why I added a new activityViewTypeId61.

Reply:
No, we don't need these two, let me remove one and add some conditions to remove the field from the authorised dashlet.